controls/V-72075.rb

```diff
@@ -71,19 +45,18 @@
 If the system is using an alternate boot loader on removable media, and
 documentation does not exist approving the alternate configuration, this is a
 finding."
-  tag "fix": "Remove alternate methods of booting the system from removable media or
-document the configuration to boot from removable media with the ISSO."
-
-  # @todo -make test more efficient/check for set root
-  grub_cfgs = command('find / -name grub.cfg').stdout.split("\n")
-  grub_cfgs.each do |cfg|
-    describe.one do
-      describe file(cfg) do
-        its('path') { should eq '/boot/grub2/grub.cfg' }
-      end
-      describe file(cfg) do
-        its('path') { should eq '/boot/efi/EFI/redhat/grub.cfg' }
-      end
-    end
+  tag "fix": "Remove alternate methods of booting the system from removable
+media or document the configuration to boot from removable media with the ISSO."
+  tag "fix_id": "F-78427r1_fix"
+
+  describe "The list of unapproved boot loader configuration files" do
+    subject {
+      command('find / -name grub.cfg -type f').stdout.chomp.split
```

Comment:
This seems to be overly aggressive and the necessary information should be able to be pulled from `grubby`